core/codenvy-hosted-platform-api-impl/src/main/java/com/codenvy/api/dao/mongo/AccountDaoImpl.java

```diff
@@ -1,408 +0,0 @@
- * herein are proprietary to Codenvy S.A.
- * and its suppliers and may be covered by U.S. and Foreign Patents,
- * patents in process, and are protected by trade secret or copyright law.
- * Dissemination of this information or reproduction of this material
- * is strictly forbidden unless prior written permission is obtained
- * from Codenvy S.A..
- */
-package com.codenvy.api.dao.mongo;
-
-import com.codenvy.api.event.user.RemoveAccountEvent;
-import com.mongodb.BasicDBList;
-import com.mongodb.BasicDBObject;
-import com.mongodb.DB;
-import com.mongodb.DBCollection;
-import com.mongodb.DBCursor;
-import com.mongodb.DBObject;
-import com.mongodb.MongoException;
-
-import org.eclipse.che.api.account.server.dao.Account;
-import org.eclipse.che.api.account.server.dao.AccountDao;
-import org.eclipse.che.api.account.server.dao.Member;
-import org.eclipse.che.api.core.ConflictException;
-import org.eclipse.che.api.core.NotFoundException;
-import org.eclipse.che.api.core.ServerException;
-import org.eclipse.che.api.core.model.workspace.Workspace;
-import org.eclipse.che.api.core.notification.EventService;
-import org.eclipse.che.api.workspace.server.spi.WorkspaceDao;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import javax.inject.Inject;
-import javax.inject.Named;
-import javax.inject.Singleton;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.LinkedList;
-import java.util.List;
-
-import static com.codenvy.api.dao.mongo.MongoUtil.asDBList;
-import static com.codenvy.api.dao.mongo.MongoUtil.asMap;
-import static com.codenvy.api.dao.mongo.MongoUtil.asStringList;
-import static java.lang.String.format;
-import static java.util.stream.Collectors.toList;
-
-/**
- * Implementation of {@link AccountDao} based on MongoDB storage.
- * <pre>
- *  Account members collection document schema:
- *
- * {
- *     "_id" : "userId...",
- *     "members" : [
- *          ...
- *          {
- *              "userId" : "userId...",
- *              "accountId" : "accountId...",
- *              "roles" : [
- *                  "role1...",
- *                  "role2..."
- *              ]
- *          }
- *          ...
- *     ]
- * }
- *
- * Account collection document schema:
- *
- * {
- *      "id" : "accountId...",
- *      "name" : "name...",
- *      "attributes" : [
- *          ...
- *          {
- *              "name" : "key...",
- *              "value" : "value..."
- *          }
- *          ...
- *      ],
- *      "workspaces" : [ "workspace123", "workspace234" ]
- * }
- *
- *
- * </pre>
- *
- * @author Max Shaposhnik
- * @author Eugene Voevodin
- * @author Alexander Garagatyi
- */
-@Singleton
-public class AccountDaoImpl implements AccountDao {
-
-    private static final Logger LOG                = LoggerFactory.getLogger(AccountDaoImpl.class);
-    private static final String ACCOUNT_COLLECTION = "organization.storage.db.account.collection";
```

Comment:
Did you prepare pull request for deployment project? 
Should we remove mongo collection for accounts?